### PR TITLE
copy edits, added resource

### PR DESCRIPTION
**Pull request by emilane**
Copy edited body content, added a resource, simplified and clarified language + standardized to American spelling ("colour" to "color").

### CODE REVIEW

**Comment by ericwbailey** [comment]
This is great, thank you! Would you mind changing the double dashes (`--`) to em dashes (`—`)? I'll merge it afterwards.

File: _posts/2017-10-18-understanding-colourblindness.md [hunk 1]
```diff
@@ -1,43 +1,37 @@
...
 
-Colour blindness is less profound than many impairments. Yet, people with colour vision problems often face restrictions on certain activities. This varies around the world. For example, Romania prohibits residents who are colour blind from driving. Yet, the USA doesn't consider colour blindness as a disability, for workplace discrimination purposes.
+Color blindness is less profound than many impairments, but people with color vision problems often face restrictions on certain activities. This varies around the world. For example, Romania prohibits residents who are color blind from driving. The USA doesn't consider color blindness a disability because of workplace discrimination purposes.
 
-I've avoided going into the different types here, as the principles are the same for each. Also, [Wikipedia's article on colour blindness](https://en.wikipedia.org/wiki/Color_blindness) has some excellent descriptions.
+Though there are multiple types of color blindness, they are the same in principle. For more information, check out [Wikipedia's article on colour blindness](https://en.wikipedia.org/wiki/Color_blindness), which has some excellent descriptions.
```
Comment (scottaohara) [inline _posts/2017-10-18-understanding-colourblindness.md]: to match other instances:
Wikipedia's article on **colour** blindness => Wikipedia's article on **color** blindness

Comment (emilane) [reply _posts/2017-10-18-understanding-colourblindness.md]: @scottaohara @ericwbailey Thank you for catching those!

File: _posts/2017-10-18-understanding-colourblindness.md [hunk 2]
```diff
@@ -32,6 +32,6 @@ There are a number of online tools to help those without color blindness visuali
 
 - [Colorblind Color + Contrast Picker](http://colorsafe.co)
 - [Colorblind web page filter by Toptal](https://www.toptal.com/designers/colorfilter)
-- [I want to see like the colour blind Chrome extension](https://chrome.google.com/webstore/detail/i-want-to-see-like-the-co/jebeedfnielkcjlcokhiobodkjjpbjia)
+- [I want to see like the color blind Chrome extension](https://chrome.google.com/webstore/detail/i-want-to-see-like-the-co/jebeedfnielkcjlcokhiobodkjjpbjia)
```
Comment (scottaohara) [inline _posts/2017-10-18-understanding-colourblindness.md]: i noticed this one as well, but following the link the extension is actually titled "I want to see like the colour blind" so it probably should stay as the original spelling

File: _posts/2017-10-18-understanding-colourblindness.md [hunk 3]
```diff
@@ -1,43 +1,37 @@
 ---
 layout: post
-title: "Understanding colourblindness"
-description: "What is colour blindness, and how can we design with it in mind?"
+title: "Understanding colorblindness"
+description: "What is color blindness, and how can we design with it in mind?"
 author: matthew_higgins
```
Comment (scottaohara) [inline _posts/2017-10-18-understanding-colourblindness.md]: one last thing, Emily.

You've done some good work here and you should get credit for it.  Please add:
`updated_by: emily_lane` on a new line after "author"

and also update the [authors yml file](https://github.com/a11yproject/a11yproject.com/blob/gh-pages/_data/authors.yml) with your information (using "emily_lane" to match the line I mentioned above") and commit to this PR.

Sorry, I didn't notice this in my previous reviews.

**Comment by ericwbailey** [comment]
Merged with the addition of https://github.com/a11yproject/a11yproject.com/pull/646

Thank you again for your contribution, @emilane!

**Comment by emilane** [comment]
Thanks @ericwbailey! I'm not the most familiar with Github, and thank you for your patience with me!

**Pull request merged**